instrumentation/jms-1.1/src/test/groovy/JMS1Test.groovy

```diff
@@ -33,7 +33,7 @@ class JMS1Test extends AgentTestRunner {
 
   private static final Logger logger = LoggerFactory.getLogger(JMS1Test)
 
-  public static GenericContainer activemq = new GenericContainer("rmohr/activemq")
+  private static final GenericContainer BROKER = new GenericContainer("rmohr/activemq")
```

Comment:
what check requires uppercase names?

Reply:
![image](https://user-images.githubusercontent.com/1764981/95459462-99b72780-096b-11eb-929d-65fee34fe2d6.png)
spotBugs recommends this to be private (although maybe sharing it between the tests might also make sense) and once you turn it `private final` _codeNarc_ complains about the naming - I believe @trask was going to look into this.

Reply:
I believe he already did: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1335. Is your baseline fresh enough?

Reply:
probably not. I will rebase and run the tasks again, thanks